config/networkconnectivity/config.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2024 The Crossplane Authors <https://crossplane.io>
+//
+// SPDX-License-Identifier: CC0-1.0
+
+package networkconnectivity
+
+import (
+	"github.com/crossplane/upjet/pkg/config"
+)
+
+// Configure configures individual resources by adding custom
+// ResourceConfigurators.
+func Configure(p *config.Provider) {
+	p.AddResourceConfigurator("google_network_connectivity_service_connection_policy", func(r *config.Resource) {
+		config.MarkAsRequired(r.TerraformResource, "region")
```

Comment:
Why are we adding this configuration, this resource does not seem to have a region parameter.

Reply:
Copy paste error.. This should be `location`. Fixed